workflows/templates/thamos-advise-template.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: thamos-advise
+  annotations:
+    thoth-station.ninja/template-version: 0.1.0-dev
+  labels:
+    app: thoth
+    component: thamos-advise
+spec:
+  templates:
+    - name: thamos-advise
+      inputs:
+        parameters:
+          - name: "GITHUB_EVENT_TYPE"
+          - name: "GITHUB_CHECK_RUN_ID"
+          - name: "GITHUB_INSTALLATION_ID"
+          - name: "ORIGIN"
+          - name: "REVISION"
+          - name: "THOTH_HOST"
+        artifacts:
+          - name: repository
+            path: /mnt/inputs/artifacts/repository
+            git:
+              repo: "{{inputs.parameters.ORIGIN}}"
+              revision: "{{inputs.parameters.REVISION}}"
+      script:
+        command:
+          - python3
+        name: thamos-advise
+        image: "quay.io/thoth-station/thamos:v0.8.0-dev"
+        source: |
+          import os
+          import json
+          from pathlib import Path
+          from thamos.lib import advise_here
+          from thamos.config import config
+
+          os.chdir("/mnt/inputs/artifacts/repository")
```

Comment:
I think we need to chdir into the clone git repo too @pacospace

Reply:
+1, you can also use our helper:

https://github.com/thoth-station/common/blob/5d24f258d2a1c1b569268edd8fad3c431ae77ca9/thoth/common/helpers.py#L42-L44

```python

from thoth.common import cwd

with cwd("/mnt/inputs/artifacts/repository"):
   # ...
```

Reply:
The cloned repository will be extracted into `/mnt/inputs/artifacts/repository`, no need to chdir into `.../repository/<clone-repository>`

Reply:
> The cloned repository will be extracted into `/mnt/inputs/artifacts/repository`, no need to chdir into `.../repository/<clone-repository>`

It's needed for `advise_here` method, that loads all the necessary files from the local directory and treats the current working directory as project root.